modules/users/server/controllers/users.authentication.server.controller.js

```diff
@@ -84,6 +84,19 @@ exports.signup = function (req, res) {
 
     },
 
+    // Report how did you hear about us to stats
+    function (user, done) {
+      statService.stat({
```

Comment:
I believe `stats` is not a good place to store strings like `freetext`.
[Documentation](https://github.com/Trustroots/trustroots/blob/master/modules/stats/server/services/stats.server.service.js#L3-L73) excerpt below:
```js
/**
 * Stats
 *
 * We support 3 types of statistics (stats).
 *
 * The first is count. A count is a single event that happens. For example, a
 * user logs in, a new member signs up, a message is sent, etc. These stats are
 * typically aggregated with functions like `SUM()`.
 *
 * The second is value. A value is the measurement of something at a point in
 * time. For example, how many members we have, how many items are in a queue,
 * how long it took for a user to reply to a message, etc. These stats are
 * typically aggregated with functions like `MEAN()`.
 *
 * The third is a more complex combination of the two. It's designed to be
 * implementation agnostic, so we can record stats in a standardised way and
 * push them to arbitrary backends. It can contain a combination of both counts
 * and values.
 *
 * Each of these stat types is supported by a function of the same name
 * (`count()`, `value()`, `stat()`). Usage is documented below.
 */
```
Stats want mainly numbers.
Perhaps meta would work:
```
{
  namespace: ...
  meta: {
    freetext: 'a fish told me'
  }
}
```
but storing them in influx seems senseless to me. Grafana won't be happy with displaying freetext.

If they need to be anonymous, we'll want to create another collection. If not, mongoose's user model could be an acceptable place.

Reply:
No it's actually better if they are not anonymous. I'd like to be able to have a (real, non-automated) follow up conversation with signups.
So we better store it in mongo then - and I don't mind if the data disappears from there at some point. It's just to know what to focus on in terms of growth.